ansible/deploy-proteus-frontend.yml

```diff
@@ -0,0 +1,26 @@
+---
+- hosts: proteus.test.ooni.io
+  vars:
+    proteus_frontend_https_port: 443
+    proteus_events_url: "https://events.proteus.test.ooni.io"
+    proteus_registry_url: "https://registry.proteus.test.ooni.io"
+    proteus_version: "v0.1.0-beta.4"
```

Comment:
@hellais for uniformity, here you may want to _bump_ to the latest version (AFAICT, nothing changed).

From my notes, I also noticed two bugs in the frontend that I was not able to fix:

- the view of message tasks to be sent to users triggered a JavaScript exception when no tasks were registered

- the list of registered probes and message tasks does not seem to be sorted according to time.

I guess these comments can be translated into proper tickets for proteus.